src/components/SearchCard.js

```diff
@@ -0,0 +1,215 @@
+import throttle from "lodash/throttle";
+import React, { useMemo, useState } from "react";
+import { constants } from "../util/helpers";
+const { Input, Card, Button, Result, Select } = require("antd");
+const { Option } = Select;
+const defaultNetwork = constants.CONTRACT_NETWORKS[0];
+
+export const SearchCard = ({ session }) => {
+  const [loading, setLoading] = useState(false);
+  const [installing, setInstalling] = useState(false);
+  const [success, setSuccess] = useState(false);
+  const [contract, setContract] = useState("");
+  const [error, setError] = useState("");
+  const [defs, setDefs] = useState([]);
+  const [network, setNetwork] = useState(defaultNetwork.value);
+
+  const resetState = () => {
```

Comment:
This name is a bit confusing. At first read I thought you would be clearing out all state variables, including network. Perhaps something like `resetData`? Idk maybe there isn't a better name.